po/Makefile.am

```diff
@@ -103,24 +99,6 @@ po/cockpit.pot: po/cockpit.c.pot po/cockpit.html.pot po/cockpit.js.pot po/cockpi
 	$(AM_V_GEN) $(MKDIR_P) $(dir $@) && \
 	msgcat --sort-output --output-file=$@ $^
 
-# Actually update the old translations with new translatable text
-update-po: po/cockpit.pot
```

Comment:
About this specific rule: I'm not really sure what the point of this is in our weblate-enabled, but I think it's useless.  That happens right before we copy in the new .po files, so the output of this command is directly overwritten.

Reply:
Right, we used to do an actual msgmerge, to avoid shipping obsolete translations. But it's not that much overhead, weblate should already filter them out and only keeps them in commented-out stanzas. So I agree that msgmerge can go now.